src/Tribe/Service_Providers/Context.php

```diff
@@ -289,7 +318,8 @@ static function ( $post_type ) {
 					'read' => [
 						Tribe__Context::FUNC          => [
 							static function () {
-								return ! empty( tribe_get_request_var( TEC::POSTTYPE, false ) ) ?: Tribe__Context::NOT_FOUND;
+								return ! empty( tribe_get_request_var( TEC::POSTTYPE,
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: 8e261c886b7aeeca65287baa31c06e75</sub>